### ISSUE TITLE
Better Accessibility for blind users during voting

### ISSUE THREAD

**Issue by Jonathan-Matthew** [OpenSlides/openslides-client#4397]
**Current behaviour**
At our last conference we had a blind user.
He used OpenSlides in Safari with iPhone 12 (IOS 18.1).
He used the integrated VoiceOver-Screenreader and the Fokus Blue 5. Gen Braille display.
He complained that during voting the description of the options were besides, over or under the buttons. So he wasn't sure from his screenreader which text belonged to which descriptions.

**Wanted behavior**
Screenreaders should be able to clearly commuicate which button stands for what option
